US-EPA-CAMD/easey-monitor-plan-api
# Fix/5958 duct waf typo

## Main Changes

- Corrected a typo that assigned the wrong field from imported data.

## Steps to Test

1. Check out any Monitor Plan, then open it and select the section _Rectangular Duct WAFs_.
2. Click the button **Create Rectangular Duct WAF**, and enter the following data:

![screenshot](https://github.com/US-EPA-CAMD/easey-monitor-plan-api/assets/104089016/d5ffc1bb-ce02-4cd2-9c29-5d7d5496abc7)

3. Confirm the item is created.
4. Click **Export Data**, then re-import the JSON file.
5. Open the row for editing and confirm the value for _Number of Traverse Points WAF_ is as it was originally input (16 in the screenshot). Previously, it would take the value of _Number of Traverse Points Reference_.

## Note

I ran `yarn run format` on the repository. See line 59 of [`src/duct-waf-workspace/duct-waf.service.ts`](https://github.com/US-EPA-CAMD/easey-monitor-plan-api/compare/fix/5958-duct_waf_typo?expand=1#diff-b0dc2ecb35f9bedbe492857f673f0ff1802c7dc5669021bde5aba99679af6145) for the only change.
